<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Event Planner</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>761</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="hlMainView">
     <item>
      <widget class="QStackedWidget" name="vStackedWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="Users">
        <widget class="QWidget" name="verticalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>301</width>
           <height>261</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="bCustomer">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Customer</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bSpeaker">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Speaker</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bParticipant">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Participant</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bManager">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Manager</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="Orders">
        <widget class="QWidget" name="verticalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>620</x>
           <y>-1</y>
           <width>141</width>
           <height>421</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="vlNewOrder">
          <item>
           <widget class="QPushButton" name="bNewOrder">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>New Order</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>621</width>
           <height>421</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="vlOrderList"/>
        </widget>
       </widget>
       <widget class="QWidget" name="Order"/>
       <widget class="QWidget" name="Events"/>
       <widget class="QWidget" name="Options">
        <widget class="QWidget" name="verticalLayoutWidget_5">
         <property name="geometry">
          <rect>
           <x>-1</x>
           <y>-1</y>
           <width>241</width>
           <height>341</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="vlOptionsLeft"/>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget_6">
         <property name="geometry">
          <rect>
           <x>239</x>
           <y>-1</y>
           <width>251</width>
           <height>341</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="vlOptionCenter"/>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget_7">
         <property name="geometry">
          <rect>
           <x>489</x>
           <y>-1</y>
           <width>271</width>
           <height>341</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="vlOptionRight"/>
        </widget>
        <widget class="QWidget" name="horizontalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>-1</x>
           <y>340</y>
           <width>761</width>
           <height>80</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="hlOptionBottom"/>
        </widget>
       </widget>
       <widget class="QWidget" name="Halls">
        <widget class="QWidget" name="verticalLayoutWidget_4">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>-10</y>
           <width>761</width>
           <height>431</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="vlHalls"/>
        </widget>
       </widget>
       <widget class="QWidget" name="Hall"/>
       <widget class="QWidget" name="Participants"/>
       <widget class="QWidget" name="Calendar">
        <widget class="QWidget" name="gridLayoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>761</width>
           <height>421</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="glCalendar"/>
        </widget>
       </widget>
       <widget class="QWidget" name="page_2"/>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>766</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
